deployment_files/.env.j2

```diff
@@ -1,4 +1,3 @@
-FLASK_ENV=production
 TRELLO_KEY=1c9ef7ff8b92a0f091bf399a9e167435
```

Comment:
This might not be secret but still limits your app to only work with your specific Trello account, so it might be nice to also make this variable